mapserver.h

```diff
@@ -1487,6 +1496,17 @@ The :ref:`REFERENCE <reference>` object
   /*                             scalebarObj                              */
   /************************************************************************/
 
+  #define MS_SCALEBAR_INTERVALS_MIN 1
+  #define MS_SCALEBAR_INTERVALS_MAX 100
+
+  #define MS_SCALEBAR_WIDTH_MIN 5
```

Comment:
Docs to udpate at https://mapserver.org/mapfile/scalebar.html#mapfile-scalebar-size

Reply:
Yes, once values are confirmed.